docs/workflow/testing/libraries/testing-android.md

```diff
@@ -154,3 +154,29 @@ The emulator can be launched with a variety of options. Run `emulator -help` to
 ### Debugging the native runtime code using Android Studio
 
 See [Debugging Android](../../debugging/mono/android-debugging.md)
+
+## Upgrading the Android NDK Version in CI Pipelines
+
+The Android NDK LTS version is released annually. CI pipelines retrieve the NDK version from Docker images hosted in the [dotnet-buildtools-prereqs-docker](https://github.com/dotnet/dotnet-buildtools-prereqs-docker) repository.
```

Comment:
It would be good to cover:

- Which month (or quarter) we expect this in and if adopting RCs is important.
- What we need to do in `main` vs `release/` and if the MAUI support timeframe gives us a degree of freedom in `release/`.